docs/dev_ops/release_procedure.md

```diff
@@ -0,0 +1,20 @@
+# Releasing in Modalitites
+This tutorial describes the procedure to release a new version of the Modalitites package.
+
+## Release Types
+We follow the release types as defined by [Semantic Versioning](https://semver.org/). The version number is defined as `MAJOR.MINOR.PATCH` where:
+- `MAJOR` is incremented when you make incompatible API changes,
+- `MINOR` is incremented when you add functionality in a backwards-compatible manner, and
+- `PATCH` is incremented when you make backwards-compatible bug fixes.
+
+
+## Releasing a new Modalitites version
+1. Update the version number in the pyproject.toml file.
+2. Commit the version bump following the convention `git commit --no-verify -m "<version number>"`.
+   The `--no-verify` flag is used to skip the pre-commit hooks.
+3. Run `git push` to push the changes to the remote repository.
+4. Make sure that all the tests pass in the main branch.
+5. Tag the commit with the version number following the convention `git tag <version number>`.
```

Comment:
Just a note: we used `v<version number>` for our v0.2.2 tests. But maybe it's better / easier without a "v".